src/integrations/admin/social-templates-integration.php

```diff
@@ -48,9 +49,7 @@ class Social_Templates_Integration implements Integration_Interface {
 	 * Social_Templates_Integration constructor.
 	 */
 	public function __construct() {
-		$this->recommended_replace_vars     = new WPSEO_Admin_Recommended_Replace_Vars();
-		$this->editor_specific_replace_vars = new WPSEO_Admin_Editor_Specific_Replace_Vars();
-		$this->group                        = 'global-templates';
+		$this->group = 'global-templates';
```

Comment:
I would just assign this when declaring the property so no constructor is needed.